buildflow/api/__init__.py

```diff
@@ -1,20 +1,26 @@
 # flake8: noqa
-from .io import SinkType, SourceType, Pullable, Pushable
```

Comment:
don't need this in this PR. but we should setup `isort` to automatically do this for us

Reply:
+1